src/fhirConfig.ts

```diff
@@ -52,6 +53,7 @@ export interface Profile {
     systemSearch: Search;
     systemHistory: History;
     bundle: Bundle;
+    export?: BulkDataAccess;

```

Comment:
We've used stubs in the past to force the client to explicitly declare they are not supporting certain code, but that means export will need to be added to the systemOperation list -- thoughts?

Reply:
Export isn't one of the [SystemOperation](https://www.hl7.org/fhir/valueset-system-restful-interaction.html) as defined by FHIR. Therefore, I didn't think we should add 'export' as a System Operation. I've added an extra [field](https://github.com/awslabs/fhir-works-on-aws-interface/blob/9dce16b7027a9dadc2af08d456cad4a384427551/src/authorization.ts#L16) to `AuthorizationRequest` called `exportType`. Therefore when we [parse](https://github.com/awslabs/fhir-works-on-aws-interface/blob/59ad75f64eab555ecc3ddb58dbcd5da35de9380d/src/utilities.ts#L81) a request we can determine if it's an export request.

If a client want to use stub to explicitly declare they're not supporting export, they can implement the `BulkAccess` interface, and then throw a "method not implemented" error. Is that what you mean by using stubs to explicitly declare code they don't support.

Reply:
Yeah one thing I am not sure of though is if they implement the stud how will the router know to support the 'export' route or not. Currently we are using the defined operations